### PR TITLE
G0 5510 dashboard scroll issue

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

**Comment by rohit-walkover** [comment]
@shalinee-walkover resolve conflicts

File: apps/web-giddh/src/app/home/components/profit-loss/profit-loss.component.html [hunk 1]
```diff
@@ -25,36 +25,34 @@
...
-                <perfect-scrollbar [scrollIndicators]="true">
                     <chart [options]="profitLossChart" *ngIf="dataFound" #currentRatio></chart>
                     <div class="noDataAvailable" *ngIf="!dataFound">
                         <img src="{{imgPath}}noDataChart.svg" alt="no data found">
                     </div>
                     <div class="clearfix mrB2">
                         <div class="">
-                            <span class="mrB1 dueAmount">Net Profit/Loss</span>
-                            <span class="total_amt pull-right">
+                            <span class="mrB1 due-amount">Net Profit/Loss</span>
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/home/components/profit-loss/profit-loss.component.html:34]: use mrB1 to  mb-1

File: apps/web-giddh/src/app/home/components/profit-loss/profit-loss.component.scss [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    display: block;
+    line-height: 1;
+}
+
+.dashboard-filter .btn-group span {
+    vertical-align: middle;
+    color: $dark-gray;
+}
+
+.underline {
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/home/components/profit-loss/profit-loss.component.scss:30]: underline class is already created pls remove it

File: apps/web-giddh/src/app/home/components/total-overdues/total-overdues-chart.component.html [hunk 3]
```diff
@@ -25,15 +25,14 @@
 				</div>
 			</div>
 			<div class="chart-div" google-chart chart="chartData" *ngIf="!requestInFlight">
-				<perfect-scrollbar [scrollIndicators]="true">
 					<chart [options]="totaloverDueChart" *ngIf="dataFound" #currentRatio></chart>
 					<div class="noDataAvailable" *ngIf="!dataFound">
 						<img src="{{imgPath}}noDataChart.svg" alt="no data found">
 					</div>
 					<div class="clearfix mrB2">
 						<div class="">
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/home/components/total-overdues/total-overdues-chart.component.html:33]: Blank div

File: apps/web-giddh/src/app/home/components/total-overdues/total-overdues-chart.component.scss [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    display: block;
+    line-height: 1;
+}
+
+.dashboard-filter .btn-group span {
+    vertical-align: middle;
+    color: $dark-gray;
+}
+
+.underline {
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/home/components/total-overdues/total-overdues-chart.component.scss:30]: underline class already using the remove it

File: apps/web-giddh/src/app/reports/components/columnar-report-component/columnar.report.component.html [hunk 5]
```diff
@@ -8,29 +8,43 @@
...
             </div>
-            <div class="col-md-2 col-sm-6 mb-2 clear-both">
+        </div>
+
+        <div class="row">
+            <div class="col-sm-3 col-xs-12 mb-2 clear-both">
                 <button class="btn btn-pink " [disabled]="!isFormValid() || isLoading" (click)="exportReport();">Export</button>
             </div>
         </div>
+
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/reports/components/columnar-report-component/columnar.report.component.html:48]: remove blank space

File: apps/web-giddh/src/app/home/components/profit-loss/profit-loss.component.scss (hunk 2 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/home/components/profit-loss/profit-loss.component.scss:30]: I am not write this new css, just separating css from TS file

File: apps/web-giddh/src/app/home/components/total-overdues/total-overdues-chart.component.html (hunk 3 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/home/components/total-overdues/total-overdues-chart.component.html:33]: Is blank class is in green color or red?

File: apps/web-giddh/src/app/home/components/total-overdues/total-overdues-chart.component.scss (hunk 4 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/home/components/total-overdues/total-overdues-chart.component.scss:30]: I am not write this new css, just separating css from TS file

File: apps/web-giddh/src/app/home/components/profit-loss/profit-loss.component.html (hunk 1 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/home/components/profit-loss/profit-loss.component.html:34]: CHeck please it's double green and i am worked on Dark green. Hope you understand

**Pull request merged**